src/Routes/SeasonsController.ts

```diff
@@ -16,7 +25,24 @@ export class SeasonsController extends BaseController {
                 BaseController._OnError(ex, res);
             }
         });
-
+/**
+ * @swagger
+ *
+ * /[seasonId]:
+ *   get:
+ *     description: Returns a list of Structs in a Season
+ *     produces:
```

Comment:
Can we show what the structure of the json output is?  If so, lets add it here and in the seasons/ route as well

Reply:
Yeah, we can. I’m unsure of the JSON structure because I don’t have knowledge of the database schema.

Reply:
We don't want to expose the db schema to users.  That's just implementation details.  What I want is to show them the json response, which is what their client would consume.  If you're unsure what that looks like, you can see it here:
https://hazzat-api-ppe.azurewebsites.net/seasons
https://hazzat-api-ppe.azurewebsites.net/seasons/12

Reply:
done